osu.Game/Overlays/SettingsPanel.cs

```diff
@@ -86,69 +94,48 @@ private void load()
                         Colour = OsuColour.Gray(0.05f),
                         Alpha = 1,
                     },
-                    SectionsContainer = new SettingsSectionsContainer
+                    loading = new LoadingLayer
                     {
-                        Masking = true,
-                        RelativeSizeAxes = Axes.Both,
-                        ExpandableHeader = CreateHeader(),
-                        FixedHeader = searchTextBox = new SeekLimitedSearchTextBox
-                        {
-                            RelativeSizeAxes = Axes.X,
-                            Origin = Anchor.TopCentre,
-                            Anchor = Anchor.TopCentre,
-                            Width = 0.95f,
-                            Margin = new MarginPadding
-                            {
-                                Top = 20,
-                                Bottom = 20
-                            },
-                        },
-                        Footer = CreateFooter()
-                    },
+                        State = { Value = Visibility.Visible }
+                    }
                 }
             };
 
+            Add(SectionsContainer = new SettingsSectionsContainer
+            {
+                Masking = true,
+                RelativeSizeAxes = Axes.Both,
+                ExpandableHeader = CreateHeader(),
+                FixedHeader = searchTextBox = new SeekLimitedSearchTextBox
+                {
+                    RelativeSizeAxes = Axes.X,
+                    Origin = Anchor.TopCentre,
+                    Anchor = Anchor.TopCentre,
+                    Width = 0.95f,
+                    Margin = new MarginPadding
+                    {
+                        Top = 20,
+                        Bottom = 20
+                    },
+                },
```

Comment:
This search textbox remains visible even when the settings is loading, I think it'll be best to have it hidden until the overlay is loaded.

Here's how it looks with the search textbox hidden until load complete:

https://user-images.githubusercontent.com/22781491/129663328-3136cc81-6ab4-4925-8d5d-574ad5e7712c.mp4

Reply:
I dunno. I intentionally left it visible so a user could start searching before the content is finished loading. Seems like a better UX to me?

Reply:
Hmm, right, nevermind that.